package_manifest.yaml

```diff
@@ -248,14 +240,14 @@ foreman_core_packages:
     rubygem-net-ping: {}
     rubygem-net-scp: {}
     rubygem-net-ssh:
-      releasers:
-        - koji-foreman
-        - koji-foreman-plugins
+      koji_tags:
+        - "{{ all_tags['foreman-el8'] }}"
+        - "{{ all_tags['foreman-plugins-el8'] }}"
     rubygem-net_http_unix: {}
     rubygem-netrc:
-      releasers:
-        - koji-foreman
-        - koji-foreman-plugins
+      koji_tags:
+        - "{{ all_tags['foreman-el8'] }}"
+        - "{{ all_tags['foreman-plugins-el8'] }}"
```

Comment:
Why do we release these to both foreman and foreman-plugins?

Reply:
Something something SCl...goes looking.... "Also build rubygem-net-ssh for non-SCL " so I think this means we can stop this.